src/amo/withExperiment.js

```diff
@@ -215,6 +281,11 @@ export const withExperiment =
         if (cleanupNeeded || addExperimentToCookie) {
           cookies.set(EXPERIMENT_COOKIE_NAME, experimentsToStore, cookieConfig);
         }
+
+        // Remove the stored variant from the Redux store.
```

Comment:
Ok but why?

Reply:
Tbph, I don't think we actually need to, based on how the code works, but it just feels safer to me to remove it after we have finished "using" it. Then we don't risk doing something "again" that really should only be done once. We're only dispatching this action if there was something in the store, so it won't get called all the time, unnecessarily, and I think it's a good practice to keep our state in a "clean" state.

Reply:
I removed this.